nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/main/java/org/nuxeo/ecm/platform/ec/notification/service/NotificationService.java

```diff
@@ -548,9 +549,9 @@ public void setEmailHelper(EmailHelper emailHelper) {
 
     @Override
     public List<DocumentModel> getSubscribedDocuments(String prefixedPrincipalName) {
-        String nxql = String.format("SELECT * FROM Document WHERE ecm:mixinType ='Notifiable' "
-                + "AND ecm:isCheckedInVersion = 0 "
-                + "AND notif:notifications/*/subscribers/* = '%s'", prefixedPrincipalName);
+        String nxql = String.format(
+                "SELECT * FROM Document WHERE ecm:mixinType ='Notifiable' " + "AND ecm:isCheckedInVersion = 0 "
```

Comment:
Could you please replace the `Notifiable` part of the string with `SubscriptionAdapter.NOTIFIABLE_FACET`?